packages/app/src-tauri/src/commands/ftp.rs

```diff
@@ -179,9 +170,7 @@ mod test {
 
         // Hopefully the server is up =D
         sleep(Duration::from_secs(2)).await;
-        assert!(ftp_upload(payload)
-            .await
-            .is_err_and(|err| err == "Failed to connect to server"));
```

Comment:
How can we track the specific errors like:
- Failed to connect to server
- Failed to login to server
- Failed to open file